build_from.py

```diff
@@ -14,31 +17,21 @@
 from regparser.diff import treediff
 from regparser.builder import Builder, LayerCacheAggregator
 
+pyparsing.ParserElement.enablePackrat()
```

Comment:
Be very careful with packrat -- last I checked, using packrat caused tests to fail